README.md

```diff
@@ -182,7 +182,15 @@ cd aframe && npm install  # Install dependencies.
 npm start  # Start the local development server.
 ```
 
-And open in your browser **[http://localhost:9000](http://localhost:9000)**.
+And open in your browser [http://localhost:8080](http://localhost:8080).
+
+With https:
+
+```sh
+npm run start:https
+```
+
+And open in your browser [https://localhost:8080](https://localhost:8080).
```

Comment:
Perhaps a good idea to add that the certificate is self-signed and the browser will show a warning screen when (first) accessing. Can imagine people unfamiliar with the details will think something is wrong at first.

Reply:
Good idea, I rewrote that, better now?

Reply:
Yes, nice that it now also states when/why you'd want https.